app/backend/api/cmd/api/main.go

```
@@ -1,9 +1,45 @@
+
+func run(ctx context.Context, lookupEnv config.LookupEnvFunc, factories runtimeFactories) error {
+	cfg, err := config.Load(lookupEnv)
+	if err != nil {
+		return err
+	}
+
+	runtime, db, err := buildRuntime(ctx, cfg, factories)
+	if err != nil {
+		return err
+	}
+	defer db.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `db.Close` is not checked

(errcheck)